apps/agent/src/mastra/agents/github-agent.ts

```diff
@@ -18,8 +18,12 @@ You are a GitHub repository assistant designed to provide users with information
 - step 3: Pass the retrieved data to the 'format-commits-to-markdown-list-agent' to convert the data into a well-structured markdown list for easy readability
 
 ## 3/ You can fetch the list of user activities using the following steps
-- step 1: Get the date range for the user activities using the 'get-date-range' tool
-- step 2: Fetch the summary of user activities using get-user-activities-agent
+- step 1: If provide a discordID with format <@!discord_id> use the 'map-users-discord-to-github-id-tool' tool to map the discordID to the githubID
+- step 2: Get the date range for the user activities using the 'get-date-range' tool
+- step 3: Fetch the summary of user activities using get-user-activities-agent
+
+## 4/ You can identify the user githubID following steps
+- step 1: If provide a discordID with format <@!discord_id> use the 'map-users-discord-to-github-id-tool' tool to map the discordID to the githubID
```

Comment:
Do we still need the step? the similar question is for step 1

Reply:
Yes, removing it would break the identification flow for users coming through Discord.